lib/plugins/blocks.js

```diff
@@ -14,13 +14,13 @@ const paintingFaceToVec = [
   new Vec3(1, 0, 0)
 ]
 
-function inject (bot, { version }) {
+function inject (bot, { version, storageBuilder }) {
   const nbt = require('prismarine-nbt')
   const Block = require('prismarine-block')(version)
   const Chunk = require('prismarine-chunk')(version)
   const ChatMessage = require('prismarine-chat')(version)
   const World = require('prismarine-world')(version)
-  bot.world = new World().sync
+  bot.world = new World(null, storageBuilder ? storageBuilder({ version: bot.version, worldName: 'overworld' }) : null).sync
```

Comment:
What happen if we move to nether or the end ? Should a single World instance deal with all dimensions ?

Reply:
That's in the respawn line in blocks.js, a new world instance is created. worldName will be nether in this case
That's why storageBuilder is a function, so you can call it several time.
One world should handle only one dimension imho

(but indeed I actually hardcoded overworld in blocks.js, will add at least nether)